src/components/LoginPasswordForm.vue

```diff
@@ -27,7 +27,7 @@
       <v-col cols="12">
         <slot name="button">
           <v-btn
-            :disabled="!validForm || disabledButton"
+            :disabled="validForm || disabledButton"
```

Comment:
So the login button will be disabled when the form is VALID 🤨?
You should figure out why the `validForm` is `false` when it should be `true`.